tests/io/qchem/test_outputs.py

```diff
@@ -523,8 +524,10 @@ def test_nbo_3_c(self):
         assert perturb_ene[0]["perturbation energy"][3209] == 3.94
 
     def test_qchem_6_1_1(self):
-        data = QCOutput(f"{TEST_FILES_DIR}/qchem/6.1.1.wb97xv.out.gz").data
-        assert data["final_energy"] == -76.43205015
+        qc_out = QCOutput(f"{TEST_FILES_DIR}/qchem/6.1.1.wb97xv.out.gz")
+        assert qc_out.data["final_energy"] == -76.43205015
+        n_vals = sum(1 for val in qc_out.data.values() if val is not None)
```

Comment:
@janosh --- am I understanding correctly that if new properties are added to the `.data` attribute that this will fail? That seems like it'd be annoying from a CI perspective. For instance, there are definitely others missing that will need to be patched in future PRs.

Reply:
that's correct. i was thinking it would be good to force future PRs to look at that test and add a line for any new property